backend/src/main/kotlin/app/ehrenamtskarte/backend/auth/webservice/schema/ResetPasswordMutationService.kt

```diff
@@ -53,6 +53,9 @@ class ResetPasswordMutationService {
             Sie können Ihr Passwort unter dem folgenden Link zurücksetzen:
             $administrationBaseUrl/reset-password/${URLEncoder.encode(key, StandardCharsets.UTF_8)}
             
+            Falls ihr Browser den Link nicht korrekt auflösen kann, kopieren Sie bitte folgenden Link in Ihren Browser:
+            $administrationBaseUrl/reset-password/$key
+            
```

Comment:
Let's not do this. The above link is always the correct one. Not URL encoding $key produces an invalid URL, and it was okay, that Outlook disrespected those = chars.
Adding this comment would just add confusion imo.
```suggestion
```